lib/src/screens/send/send_template_page.dart

```diff
@@ -145,7 +145,7 @@ class SendTemplatePage extends BasePage {
                                     .primaryTextTheme
                                     .headline
                                     .decorationColor),
-                            validator: sendViewModel.addressValidator,
+                            //validator: sendViewModel.addressValidator,
```

Comment:
Better to make new validator which will use address validator and domain validator (create if not exists) and keep validation for address fields.